osu.Game/Tests/Visual/SkinnableTestScene.cs

```diff
@@ -152,25 +153,38 @@ private class TestLegacySkin : LegacySkin
         {
             private readonly bool extrapolateAnimations;
 
+            private readonly HashSet<string> legacyFontPrefixes = new HashSet<string>();
+
             public TestLegacySkin(SkinInfo skin, IResourceStore<byte[]> storage, AudioManager audioManager, bool extrapolateAnimations)
                 : base(skin, storage, audioManager, "skin.ini")
             {
                 this.extrapolateAnimations = extrapolateAnimations;
+
+                legacyFontPrefixes.Add(GetConfig<string, string>("HitCirclePrefix")?.Value ?? "default");
+                legacyFontPrefixes.Add(GetConfig<string, string>("ScorePrefix")?.Value ?? "score");
+                legacyFontPrefixes.Add(GetConfig<string, string>("ComboPrefix")?.Value ?? "score");
```

Comment:
can't these be actual lookups rather than magic strings? `ComboPrefix` was added in this PR already

Reply:
didn't make them lookups as they're legacy settings, therefore their names shouldn't really change or something, and making them lookups may not really be worth it, as `HitCirclePrefix` is defined by `OsuSkinConfiguation` in osu!standard ruleset assembly.

Reply:
Sure, I guess this is not worth spending much time on.

Reply:
this is shocking and should not be in this PR. or in this test class alone.